packages/components/src/Link/Link.stories.tsx

```diff
@@ -0,0 +1,242 @@
+import { Story } from "@storybook/react/types-6-0";
+import React from "react";
+import Clickable from "../Clickable";
+import Heading from "../Heading";
+import CheckCircleRoundedIcon from "../Icons/CheckCircleRounded";
+import Text from "../Text";
+import Link, { LinkProps } from "./Link";
+import styles from "./Link.stories.module.css";
+
+const sizes = ["small", "medium", "large"] as const;
+const allColors = ["alert", "brand", "neutral", "success", "warning"] as const;
+const variants = ["flat", "outline", "link"] as const;
+const states = ["inactive", "hover", "focus", "active", "disabled"] as const;
+
+// For now, the UI kit only includes alert & brand "flat" buttons.
+const flatColors = ["alert", "brand"] as const;
+
+export default {
+  title: "Link",
+  component: Link,
+  argTypes: {
+    children: {
+      control: {
+        type: "text",
+      },
+    },
+    color: {
+      control: {
+        type: "radio",
+        options: allColors,
+      },
+    },
+  },
+};
+
+type Args = React.ComponentProps<typeof Link>;
+
+const Template: Story<Args> = (args) => <Link {...args} />;
+
+// TODO: Consolidate with button stories.
+
+const defaultArgs = {
+  children: "Link",
+  variant: "flat" as const,
+  size: "medium" as const,
+  color: "brand" as const,
+  href: "",
+};
+
+export const Primary = Template.bind(null);
+Primary.args = {
+  ...defaultArgs,
+};
+
+export const Secondary = Template.bind(null);
+Secondary.args = {
+  ...defaultArgs,
+  variant: "outline",
+};
+
+export const Tertiary = Template.bind(null);
+Tertiary.args = {
+  ...defaultArgs,
+  color: "neutral",
+  variant: "outline",
+};
+
+export const Destructive = Template.bind(null);
+Destructive.args = {
+  ...defaultArgs,
+  color: "alert",
+};
+
+export const PrimarySmall = Template.bind(null);
+PrimarySmall.args = {
+  ...defaultArgs,
+  size: "small",
+};
+
+export const SecondarySmall = Template.bind(null);
+SecondarySmall.args = {
+  ...defaultArgs,
+  size: "small",
+  variant: "outline",
+};
+
+export const TertiarySmall = Template.bind(null);
+TertiarySmall.args = {
+  ...defaultArgs,
+  color: "neutral",
+  size: "small",
+  variant: "outline",
+};
+
+export const LinkButton = Template.bind(null);
```

Comment:
Would suggest:
- Removing all the stories above this (they're more design guidance for when to use what style, I added them to make Zeroheight match the UI Kit)
- renaming "Link Button" to "Link" throughout

Reply:
Would you suggest this change for the `Button` component stories as well?

Reply:
showing the grids first makes sense 👍  re: `Button` component stories, maybe we can show just one link variant there and rename it `linkVariant`

Reply:
Good idea. I think I'll take another look at the stories in a separate PR so we can get the initial component out first.